examples/09-depth/generic_depth.cpp

```diff
@@ -237,6 +237,9 @@ struct __declspec(uuid("e006e162-33ac-4b9f-b10f-0e15335c7bdb")) generic_depth_de
 		depth_stencil_backup &backup = depth_stencil_backups.emplace_back();
 		backup.depth_stencil_resource = resource;
 
+		if (device->check_capability(device_caps::resolve_depth_stencil))
+			desc.texture.samples = 1; // Backup texture should either resolved or never created.
```

Comment:
I don't think this needs to be conditional, can just always force sample count to 1 and move this to the rest of those setters above (line 210).